pf/tests/testdata/smac-program/Pulumi.yaml

```diff
@@ -0,0 +1,42 @@
+name: smac-program
```

Comment:
Here's the test @iwahbe 

It's still weird how this works out. The property is passed as bool. But Configure returns it as string because it's a string internally. The functions verify they receive the right string values. 

Something in the Yaml SDK must be auto-converting  for us.